tests/integration/api/error_group_callback/test_error_group_callback_error_web.php84.php

```diff
@@ -0,0 +1,178 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+Tests newrelic_set_error_group_callback() API for Web errors.
```

Comment:
Can we add to the description or the `skipif` printout below why this test is separate for 8.4 specifically?